pennylane/labs/dla/lie_closure_dense.py

```diff
@@ -101,6 +101,12 @@ def lie_closure_dense(
 
     >>> g_ops = [qml.pauli_decompose(op) for op in g]
 
+    **Internal representation**
+
+    The input operators are converted to Hermitian matrices internally. This means
+    that we compute the operators :math:`G_\alpha` in the algebra :math:`\{iG_\alpha\}_\alpha`,
+    which itself consists of skew-Hermitian objects (commutators produce skew-Hermitian objects,
+    so Hermitian operators alone can not form an algebra).
```

Comment:
I noticed that one _could_ falsify this statement by introducing the Lie bracket $i[ ., . ]$. So just to be precise, maybe we can say this:
```suggestion
    so Hermitian operators alone can not form an algebra with the standard commutator).
```